pandas/io/common.py

```diff
@@ -176,6 +183,7 @@ def get_filepath_or_buffer(
     compression: CompressionOptions = None,
     mode: ModeVar = None,  # type: ignore[assignment]
     storage_options: StorageOptions = None,
+    headers: dict = {},
```

Comment:
Can we be more specific using Dict? What types are the keys and values?

Reply:
It's HTTP Headers, so it should be `str` for keys, and the type for value can vary I think.